web/src/components/Request.tsx

```diff
@@ -265,34 +273,107 @@ export const RequestAccessInstructions: React.FC = () => {
     request?.grant != null ? request.id : ""
   );
 
+  const [refreshInterval, setRefreshInterval] = useState(0);
+  const { data: reqData } = useUserGetRequest(
+    request?.grant != null ? request.id : "",
+
+    {
+      swr: {
+        refreshInterval: refreshInterval,
+      },
+    }
+  );
+
+  useEffect(() => {
+    if (reqData?.grant?.status !== "ACTIVE") {
+      setRefreshInterval(1000);
+    } else {
+      setRefreshInterval(0);
+    }
+  }, [reqData]);
+
   if (!data || !data.instructions) {
     return null;
   }
 
+  if (reqData?.grant?.status === "PENDING") {
+    return (
+      <Stack>
+        <Box textStyle="Body/Medium" mb={2}>
+          Granted is provisioning your access, it'll be done soon!
+        </Box>
+        <Progress size="xs" isIndeterminate />
+      </Stack>
+    );
+  }
+
   return (
     <Stack>
       <Box textStyle="Body/Medium" mb={2}>
         Access Instructions
       </Box>
       return (
       <ReactMarkdown
+        // remarkPlugins={[remarkGfm]}
+        skipHtml={false}
```

Comment:
Is `skipHtml` required? I'm concerned that if we start allowing arbitrary HTML in the setup instructions we'll expose the app to XSS vulnerabilities.

```suggestion
```